samples/sensor/hts221/sample.yaml

```diff
@@ -1,7 +1,18 @@
 sample:
   name: HTS221 Temperature and Humidity Monitor
+common:
+    tags: sensor
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: false
+      regex:
+        - "Temperature:(.*)C"
+        - "Relative Humidity:(.*)%"
+      fixture: fixture_i2c_hts221 # see documentation
```

Comment:
> see documentation.. 

what documentation?

Reply:
still please remember our voice convo, where we said this TC shall be modified not to do an expect sequence, as it is actually a ztest -- if a condition happens which would make the _printing_ impossible, then it shall raise a failure/error

Reply:
@nashif: Added documentation with my limited knowledge about harness and it's various config options. Additionally, mentioned how to and where to use fixture keyword. Kindly check.

Reply:
@inakypg: The current test that is posted here is part of "samples" that are not ztest based. So, I think regex is required as the test will not print any "PASS" or "PRIJECT EXECUTION SUCCESSFUL" statement.
However, we will not be using regex option for ztest based test cases as suggested.